server/neptune/gateway/event/check_run_handler.go

```diff
@@ -75,9 +76,12 @@ func (h *CheckRunHandler) Handle(ctx context.Context, event CheckRun) error {
 	// we only handle requested/re-requested action types
 	switch event.Action.GetType() {
 	case RequestedActionType:
-		return h.handleRequestedAction(ctx, event, rootName)
+		// wrap this in a scheduler for consistent err handling (ie. logging)
+		return h.SyncScheduler.Schedule(ctx, func(ctx context.Context) error {
```

Comment:
maybe I missed this earlier but why are we using a different schedulers for `RequestedActionType` and `ReRequestedActionType`?

Reply:
I think samra answered this offline